contrib/vagrant/provision-util.sh

```diff
@@ -81,6 +81,9 @@ os::provision::init-certs() {
   local volumes_dir="/var/lib/openshift.local.volumes"
   local cert_dir="${server_config_dir}/master"
 
+  # Clear stale configuration
+  rm -rf ${config_root}/openshift.local.*
```

Comment:
removing etcd and config directories doesn't seem reasonable here... I could see removing the stale marker after generating the master and node certs...